prow/plugins/trigger/pr.go

```diff
@@ -128,6 +136,10 @@ func trustedPullRequest(ghc githubClient, pr github.PullRequest) (bool, error) {
 }
 
 func buildAll(c client, pr github.PullRequest) error {
+
+	if err := c.GitHubClient.RemoveLabel(pr.Base.Repo.Owner.Login, pr.Base.Repo.Name, pr.Number, needsOkToTest); err != nil {
```

Comment:
This is a strange place to do the label removal. Why not remove the label in `handleIC`, where we have the list of labels on the issue as well as the comment saying "ok to test"?